template/{% if git_platform=="github.com" %}.github{% endif %}/workflows/ci.yml.jinja

```diff
@@ -50,6 +50,7 @@ jobs:
     needs: check
     if: needs.check.outputs.branch-pr == ''
     uses: ./.github/workflows/_dist.yml
+  {% if pypi %}
   {% raw %}
```

Comment:
Please can you get rid of the secrets passing here and in `_pypi.yml`, and the `with: password` section of the pypi publish action so that it is forced to use trusted publishing